### PR TITLE
Create check_polygon.py

### PR DESCRIPTION
**Pull request by sarkarghya**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by L3str4nge — APPROVED** [review]

**Comment by QuantumNovice** [comment]
@sarkarghya use `python black` on your code. Everything else is ok.

**Review by poyea — APPROVED** [review]

**Comment by poyea** [comment]
@algorithms-keeper review

File: maths/check_polygon.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+
+def check_polygon(nums: List) -> bool:
+    """
+    Takes list of possible sidelengths and determines whether a two-dimensional polygon with such sidelengths can exist.
+    Return a boolean value for the < comparison of the largest sidelength with sum of the rest.
```
Comment (poyea) [inline maths/check_polygon.py:7]:
```suggestion
    Takes list of possible side lengths and determines whether a
    two-dimensional polygon with such side lengths can exist.
    
    Returns a boolean value for the < comparison 
    of the largest side length with sum of the rest.
```

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩 Looks good!

**Pull request merged**